.gitignore

```diff
@@ -101,4 +101,3 @@ typings/
 
 # TernJS port file
 .tern-port
-package-lock.json
```

Comment:
I'm adding the package-lock to source control. This is the recommended way. It ensures that the dependencies we develop against are the exact same versions that the user will install. Let me know if you have strong feelings against this.

Reply:
Im surprised it wasn't already there, thanks for adding it.

Reply:
Well, removing it.